build.gradle

```diff
@@ -38,11 +39,12 @@ gradlePlugin {
 
     automatedPublishing = true
 
+    def releaseNotesFile = layout.projectDirectory.file('release/changes.md')
     plugins {
         commonCustomUserData {
             id = 'com.gradle.common-custom-user-data-gradle-plugin'
             displayName = 'Gradle Enterprise Common Custom User Data Gradle Plugin'
-            description = 'A Gradle plugin to capture common custom user data used for Gradle Build Scans in Gradle Enterprise'
+            description = providers.fileContents(releaseNotesFile).asText.get()
```

Comment:
It is probably still a good idea to throw a `.trim()` on here. Since you're reading the value at configuration time, it is probably fine to just append `.trim()` here rather than doing a `.map { it.trim() }`.

Reply:
Yep I missed that, thanks.